hr_timesheet_default_analytic_account/tests/test_timesheet_creation.py

```diff
@@ -1,22 +1,21 @@
-# -*- coding: utf-8 -*-
 # Copyright 2019 Coop IT Easy SCRLfs
 #   - Vincent Van Rossem <vincent@coopiteasy.be>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp.tests.common import TransactionCase
-from openerp.exceptions import Warning as UserError
+from odoo.exceptions import Warning as UserError
```

Comment:
You can import UserError directly. [Warning was kept for compatibility](https://github.com/coopiteasy/OCB/blob/a79b3109a5407c666974da9dc83420e88a69628f/odoo/exceptions.py#L37).

Reply:
thanks! done!